tools/install.sh

```diff
@@ -21,7 +21,7 @@ else
   BOLD=""
   NORMAL=""
 fi
-CHECK_ZSH_INSTALLED=$(grep /zsh$ /etc/shells | wc -l)
+CHECK_ZSH_INSTALLED=$((grep /zsh$ /etc/shells || whereis zsh) | wc -l)
```

Comment:
Might want to use some `which` logic instead of `whereis` here. The Linux version of `whereis` still produces output when it doesn't find anything.

```
$ uname -a
Linux debbie 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt11-1 (2015-05-24) x86_64 GNU/Linux
$ whereis no-such-command
no-such-command:
$ whereis no-such-command | wc -l
1 
```